frontend/express/public/javascripts/countly/countly.views.js

```diff
@@ -3328,6 +3335,55 @@ window.ManageUsersView = countlyView.extend({
         $(".change-password").off("click").on('click', function() {
             $(this).parents(".row").next().toggle();
         });
+
+        $('body').off('change').on('change', '.pp-uploader', function() {
+            $('.pp-menu-list').hide();
+            var member_id = $(this).data('member-id');
+            $(this).simpleUpload("/member/icon", {
+                data: {
+                    _csrf: countlyGlobal.csrf_token,
+                    member_image_id: member_id
+                },
+                success: function(data) {
+                    if (data) {
+                        $('.member-image-path').val(data);
+                        $('#pp-circle-' + member_id).find('span').hide();
+                        $('#pp-circle-' + member_id).css({'background-image': 'url("' + data + '?now=' + Date.now() + '")', 'background-size': '100%', 'background-position': '0 0'});
+                        if (member_id === countlyGlobal.member._id) {
+                            $('.member_image').html("");
+                            $('.member_image').css({'background-image': 'url("' + data + '?now=' + Date.now() + '")', 'background-size': '100%', 'background-position': '0 0'});
+                        }
+                    }
+                },
+                error: function() {
+                    CountlyHelpers.notify(jQuery.i18n.map["plugins.errors"]);
+                }
+            });
+        });
+
+        $('.delete-symbol').on('click', function() {
```

Comment:
What does symbol has to do with this?

Reply:
https://github.com/Countly/countly-server/pull/906/files#diff-2d40c6d737f10cb1de3e55b86434ca35R3419

Reply:
Ok, but still let's renamed to something more valid, like `delete-member-image`